src/modules/checkBox.js

```diff
@@ -0,0 +1,13 @@
+const CheckBoxesChecked = (event) => {
+  if (event.target.classList.contains('checkbox')) {
+    const list = JSON.parse(localStorage.getItem('mytodoList'));
+    list[event.target.id - 1].completed = !list[event.target.id - 1].completed;
+    event.target.parentNode.classList.toggle('newbackg');
+    event.target.nextElementSibling.classList.toggle('input-completed');
+    event.target.nextElementSibling.classList.toggle('completed-task');
+    event.target.parentNode.lastElementChild.classList.toggle('trash');
+    localStorage.setItem('mytodoList', JSON.stringify(list));
+  }
+};
```

Comment:
- [ ] Great job creating a new JavaScript file that contains methods related to the status updates (completed: true/false) and importing it as a module 🥇 . However, the task object values updated to "completed" need to be stored in local storage to ensure that the checkbox remains false even when I reload the page. Currently, the completed task becomes true again every time you reload the page. Kindly fix this 😄